include/seastar/core/semaphore.hh

```diff
@@ -466,6 +516,7 @@ public:
     /// Return ownership of all units. The semaphore will be signaled by the number of units returned.
     void return_all() noexcept {
         if (_n) {
+            _sem->sub_outstanding_units(_n);
```

Comment:
What about return_units()?

Reply:
duh, fell between the cracks. will fix.

Reply:
@bhalevy did you fix it? This conversation was never resolved.

Reply:
@nyh yes, it was fixed.
See 7 lines up